pkg/prompts/prompts.go

```diff
@@ -510,6 +518,33 @@ func (r *realPrompter) CaptureAnyList(promptStr string, options any) (any, error
 	return r.CaptureList(promptStr, strArr)
 }
 
+func (*realPrompter) CaptureEmail(promptStr string, arg any) (any, error) {
+	prompt := promptui.Prompt{
+		Label:    promptStr,
+		Validate: validateEmail,
+	}
+
+	str, err := prompt.Run()
+	if err != nil {
+		return "", err
+	}
+
+	return str, nil
+}
+
+func (*realPrompter) CaptureEmpty(promptStr string, arg any) (any, error) {
```

Comment:
probably call it CaptureString and the other, CaptureNonEmptyString ? Or something else, but would like to see a description of the captured value after Capture prefix, just Empty does not follow the other names convention

Reply:
Yep has been reported by @dasconnor as well. Already addressed.